redshift_sqlalchemy/dialect.py

```diff
@@ -90,6 +101,8 @@ def post_create_table(self, table):
                 keys = (sortkey,)
             else:
                 keys = sortkey
+            if sortkey_type == 'interleaved':
```

Comment:
Might not be for this branch, but I greatly prefer using string constants or dedicated objects for things like this, since that makes typos a lot less dangerous.